Makefile

```diff
@@ -52,6 +54,7 @@ LOCAL_DB = tmp/local.sqlite3
 # CUE imports needed for schema validation
 ATTRIBUTES_DATA_CUE = schema/attributes_data.cue
 CATEGORIES_DATA_CUE = schema/categories_data.cue
+MAPPINGS_DATA_CUE = schema/mappings_data.cue
```

Comment:
You only need this if you need the data generated as part of integrity checks (see `validations.cue`)

Reply:
without `schema/mappings_data.cue`, `cue vet` will error out 
```
mappings: field is required but not present:
    ./schema.cue:59:1
```